bundles/distribution/src/main/assembly/eclipselink.zip.xml

```diff
@@ -24,57 +24,37 @@
     <dependencySets>
         <dependencySet>
             <outputDirectory>eclipselink/jlib/</outputDirectory>
-            <outputFileNameMapping>${artifact.artifactId}.${artifact.extension}</outputFileNameMapping>
+            <outputFileNameMapping>${artifact.artifactId}${dashClassifier?}.${artifact.extension}</outputFileNameMapping>
             <includes>
                 <include>${project.parent.groupId}:eclipselink</include>
             </includes>
         </dependencySet>
-        <dependencySet>
-            <outputDirectory>eclipselink/jlib/</outputDirectory>
-            <outputFileNameMapping>${artifact.artifactId}-src.${artifact.extension}</outputFileNameMapping>
-            <includes>
-                <include>${project.parent.groupId}:eclipselink:*:sources</include>
-            </includes>
-        </dependencySet>
-        <dependencySet>
-            <outputDirectory>eclipselink/</outputDirectory>
-            <outputFileNameMapping>${artifact.artifactId}-javadocs.${artifact.extension}</outputFileNameMapping>
-            <includes>
-                <include>${project.parent.groupId}:eclipselink:*:javadoc</include>
-            </includes>
-        </dependencySet>
         <dependencySet>
             <outputDirectory>eclipselink/jlib/extension/</outputDirectory>
-            <outputFileNameMapping>${artifact.artifactId}.${artifact.extension}</outputFileNameMapping>
+            <outputFileNameMapping>${artifact.artifactId}${dashClassifier?}.${artifact.extension}</outputFileNameMapping>
             <includes>
                 <include>${project.parent.groupId}:org.eclipse.persistence.extension</include>
             </includes>
         </dependencySet>
         <dependencySet>
             <outputDirectory>eclipselink/jlib/jpa/</outputDirectory>
-            <outputFileNameMapping>${artifact.artifactId}.${artifact.extension}</outputFileNameMapping>
+            <outputFileNameMapping>${artifact.artifactId}${dashClassifier?}.${artifact.extension}</outputFileNameMapping>
             <includes>
                 <include>${project.parent.groupId}:org.eclipse.persistence.jpa.modelgen.processor</include>
                 <include>${project.parent.groupId}:org.eclipse.persistence.jpars</include>
```

Comment:
@rfelcman again, every classifier for given artifact available in project deps is added - so this covers jpars-javadoc and sources as well